resources/setups/common/base_config.json

```diff
@@ -49,7 +49,18 @@
       "enabled": true
     },
     "dot1x": {
-      "enabled": true
+      "enabled": true,
+      "interface": "eth0",
+      "radius": {
+          "secret": "SECRET",
+          "id": "02:42:ac:18:00:70",
```

Comment:
all of these except for the listen_ip seem to be the obvious defaults for running locally inside of the container?  If so, then I'd default them in the code and not here -- so this would represent explicit overrides for something outside of the conatiner.